scripts/setup-action-configs.sh

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Required environment variables
+: "${CHANGE_TYPE:? "CHANGE_TYPE environment variable must be set"}"
```

Comment:
What does the `:` at the start of the line do?

Reply:
Nothing. It's a common bash tool for when you need nothing to happen while doing something. In this scenario, I'm not performing any operation in this line, but I'd like to interpolate the value of `CHANGE_TYPE` to verify that it's populated. If it is not populated, I'd like an error thrown (`:?`) with the text `CHANGE_TYPE environment variable must be set` to make it clear that the script expects that environment variable set.

Reply:
Roger